pkg/riff/commands/handler_delete.go

```diff
@@ -71,7 +71,11 @@ func NewHandlerDeleteCommand(ctx context.Context, c *cli.Config) *cobra.Command
 		Use:   "delete",
 		Short: "delete handler(s)",
 		Long: strings.TrimSpace(`
-<todo>
+Delete one or more handlers by name or all handlers within a namespace.
+
+HTTP requests previously received by the handler will fail. Creating a new
+handler with the same name will start to receive HTTP requests previously
+delivered to the deleted handler.
```

Comment:
This is confusing. Haven't HTTP requests previously delivered to the deleted handler failed, in which case how can they be re-delivered?

Reply:
Ended up with 

```
New HTTP requests addressed to the handler will fail. A new handler created with
the same name will start to receive new HTTP requests addressed to the same
handler.
```